backend/src/files/files.controller.ts

```diff
@@ -42,4 +52,27 @@ export class FilesController {
 			statusCode: 302,
 		};
 	}
+
+	@Post("export-markdown")
+	@ApiOperation({
+		summary: "Export Markdown",
+		description: "Export Markdown to various formats",
+	})
+	@ApiBody({ type: ExportFileRequestBody })
+	@ApiResponse({ status: 200, description: "File exported successfully" })
+	async exportMarkdown(
```

Comment:
This API should be public, because not logged in users can access to this API in shared documents.